py/torch_tensorrt/dynamo/_compiler.py

```diff
@@ -788,20 +798,47 @@ def contains_metadata(gm: torch.fx.GraphModule) -> bool:
             "Some nodes do not have metadata (shape and dtype information). This could lead to problems sometimes if the graph has PyTorch and TensorRT segments."
         )
 
+    ############ TODO: testing only ############
+    use_hierarchical_partitioner = True
+    backend_priority = ["inductor", "tensorrt"]
+    backend_support_map = {
+        "inductor": {
+            "torch.ops.aten.convolution.default",
+        },
+        "tensorrt": CONVERTERS.keys(),
+    }
+    #############################################
     # Partition module into components that can be TRT-accelerated
     fast_partitioner_failed = False
     # If specified, try using the fast partitioner and fall back to the global one on failure
     if settings.use_fast_partitioner:
         try:
-            logger.info("Partitioning the graph via the fast partitioner")
-            partitioned_module, supported_ops = partitioning.fast_partition(
-                gm,
-                verbose=settings.debug,
-                min_block_size=settings.min_block_size,
-                torch_executed_ops=settings.torch_executed_ops,
-                require_full_compilation=settings.require_full_compilation,
-                skip_fusion=(num_supported_ops == total_ops),
-            )
+            if use_hierarchical_partitioner:
```

Comment:
Should this be removed as well or should we just tie it to a set of hidden settings?